packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -855,6 +855,9 @@ export interface CallCompositeStrings {
     peoplePaneTitle: string;
     permissionToReachTargetParticipantNotAllowedMoreDetails?: string;
     permissionToReachTargetParticipantNotAllowedTitle?: string;
+    pinParticipantForMe: string;
```

Comment:
Do you guys think we should rename this to `pinParticipantMenuLabel` because we have `removeMenuLabel` and `startSpotlightMenuLabel` here?